### ISSUE TITLE
xml validator (xml validator + schema parser) .dll error for registering into regsvr32.exe

### ISSUE THREAD

**Issue by ghost** [dineshkummarc/DashboardX#44]
xml validator (xml validator + schema parser)

the problem is that when ever i used to run the code i have to run a xml.dll file using run menu (regsvr32.exe) on my system and it creates problem for me to use code on clientside if i deploy the code on any of the server.

how to resolve the problem of using regsvr32.exe each and every time.

**Comment by dineshkummarc** [comment]
added this code :
uses ShellApi;
// ...
ShellExecute(0, 'open', 'regsvr32.exe', '/s '+ PChar(FileName), nil, SW_HIDE);

but its not working for me.

**Comment by dineshkummarc** [comment]
using the same code for registering dll on client side :+1: 

Assembly asm = Assembly.LoadFrom(@"Path to file");
RegistrationServices regAsm = new RegistrationServices();
bool bResult = regAsm.RegisterAssembly(asm, AssemblyRegistrationFlags.SetCodeBase);

**Comment by ghost** [comment]
yeah dineshkummarc, the second one seems to be is the better option for the question asked.
